src/docs/background/off-session-payments/index.md

```diff
@@ -62,7 +65,7 @@ kicks in), the payment card might have expired, etc. It is therefore
 always important to allow for a fall-back payment path in your
 transaction process. Since only one transition from a state can be
 triggered automatically, this fall-back payment path must be defined to
-trigger upon a user action.
+trigger upon a user action, as in the example.
 
```

Comment:
TODO: Add a 'drawer' component to explain the Stripe off-session dynamic with :use-customer-default-payment-method and link to the article. It's mentioned in the image, but linking from the image is not possible

Reply:
Added